elodie/media/media.py

```diff
@@ -287,3 +294,11 @@ def get_class_by_file(cls, _file, classes):
                 return i(_file)
 
         return None
+
+    @classmethod
```

Comment:
Thanks. I'm new to python so there's probably lots of those types of things sprinkled about. Feel free to fix them as you come across them :).